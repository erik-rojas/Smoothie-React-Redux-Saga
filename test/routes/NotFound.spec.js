import React from 'react';
import { mount } from 'enzyme';

import NotFound from 'routes/NotFound';

function setup() {
  return mount(<NotFound />);
}

describe('NotFound', () => {
  const wrapper = setup();

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should redirect to home', () => {
    navigate({ pathname: '/some-page' });

    wrapper.find('Link').simulate('click');
    expect(location.pathname).toBe('/');
  });
});
