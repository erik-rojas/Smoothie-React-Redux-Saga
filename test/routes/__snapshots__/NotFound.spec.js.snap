// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render properly 1`] = `"<div class=\\"app__not-found app__route\\"><h1>404</h1><h2><a href=\\"/\\">react-redux-saga-boilerplate</a></h2></div>"`;
